backend/Controllers/SightingsController.cs

```diff
@@ -30,4 +31,42 @@ public async Task<IActionResult> Create(SightingsRequest sightingRequest)
         }
     }
 
+    [HttpDelete("sighting={sightingId}&user={userId}/delete")]
+    public async Task<IActionResult> Delete([FromRoute] int sightingId, int userId)
+    {
+        try
+        {
+            await _service.DeleteSighting(sightingId, userId);
+            return Ok();
+        }
+        catch (UnauthorizedAccessException ex)
+        {
+            return Unauthorized(ex.Message);
+        }
+        catch (Exception ex)
+        {
+            return BadRequest(ex.Message);
+        }
+
+    }
```

Comment:
I think this isn't meant to be in this PR and the PR it's from has just been approved, it'll be worth merging the delete pr into main and then merging latest main into this branch to reduce the risk of the old parts of this code ending up in main

Reply:
Done! Would you like us to take the occasion to also modify the url for delete from     [HttpDelete("sighting={sightingId}&user={userId}/delete")]
to     [HttpDelete("{sightingId}/delete&user={userId}")] ?
And/or we should raise a new ticket about this?

Reply:
That's a great idea, why don't you make that change now before you merge this PR in